docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -268,12 +269,23 @@ Note that only one reporter can write to `stdout`. All other reporters must outp
 
 ### -s \<path\>, --screenshots \<path\>
 
-Enables screenshot capturing and specifies the directory where screenshots are saved.
+Enables screenshot capturing and specifies the root directory where screenshots are saved.
 
 ```sh
 testcafe all tests/sample-fixture.js -s screenshots
 ```
 
+#### Path Patterns
+
+The captured screenshots are organized into subdirectories within the root directory. The screenshots' relative path and name are defined using the default screenshot path patterns:
+
+* `${DATE}_${TIME}\test-${TEST_INDEX}\${USERAGENT}\{$FILE_INDEX}.png` if the [quarantine mode](#-q---quarantine-mode) is disabled;
+* `${DATE}_${TIME}\test-${TEST_INDEX}\run-${QUARANTINE_ATTEMPT}\${USERAGENT}\{$FILE_INDEX}.png` if the [quarantine mode](#-q---quarantine-mode) is enabled.
+* `${DATE}_${TIME}\test-${TEST_INDEX}\${USERAGENT}\errors\{$FILE_INDEX}.png` if the [--screenshots-on-fails](#-s---screenshots-on-fails) option is specified.
+* `${DATE}_${TIME}\test-${TEST_INDEX}\run-${QUARANTINE_ATTEMPT}\${USERAGENT}\errors\{$FILE_INDEX}.png` if the [quarantine mode](#-q---quarantine-mode) and [--screenshots-on-fails](#-s---screenshots-on-fails) option are enabled.
+
+You can also specify a custom pattern using the [--screenshot-path-pattern](#-p---screenshot-path-pattern) option.
```

Comment:
You can also specify a custom pattern using the [--screenshot-path-pattern](#-p---screenshot-path-pattern) option. -> You can also use the [--screenshot-path-pattern](#-p---screenshot-path-pattern) option to specify a custom pattern.